packages/garbo/src/outfit/dropsgear.ts

```diff
@@ -364,3 +365,28 @@ function powerGlove(): Map<Item, number> {
     ],
   ]);
 }
+
+const POSSIBLE_SNEEGLEEB_DROPS = Item.all().filter(
+  (i) =>
+    i.tradeable && i.discardable && (i.inebriety || i.fullness || i.potion),
+);
+let sneegleebDropValue: number;
+const SNEEGLEEB_DROP_RATE = 0.13;
+const MAX_SNEEGLEEB_PRICE = 100_000; // arbitrary, to help avoid outliers
+function sneegleebs(): Map<Item, number> {
+  sneegleebDropValue ??=
+    sum(POSSIBLE_SNEEGLEEB_DROPS, (item) =>
+      Math.min(garboValue(item), MAX_SNEEGLEEB_PRICE),
+    ) / POSSIBLE_SNEEGLEEB_DROPS.length;
+  return new Map<Item, number>(
+    (
+      [
+        [$item`KoL Con 13 snowglobe`, sneegleebDropValue * SNEEGLEEB_DROP_RATE],
+        [
+          $item`can of mixed everything`,
+          (sneegleebDropValue * SNEEGLEEB_DROP_RATE) / 2,
+        ],
```

Comment:
```suggestion
let sneegleebBonus: number;
const SNEEGLEEB_DROP_RATE = 0.13;
const MAX_SNEEGLEEB_PRICE = 100_000; // arbitrary, to help avoid outliers
function sneegleebs(): Map<Item, number> {
  sneegleebBonus ??=
    sum(POSSIBLE_SNEEGLEEB_DROPS, (item) =>
      Math.min(garboValue(item), MAX_SNEEGLEEB_PRICE),
    ) / POSSIBLE_SNEEGLEEB_DROPS.length * SNEEGLEEB_DROP_RATE;
  return new Map<Item, number>(
    (
      [
        [$item`KoL Con 13 snowglobe`, sneegleebBonus],
        [$item`can of mixed everything`, sneegleebBonus / 2],
```
Could make it a little more concise.

Reply:
I had considered this, but hadn't thought of the right variable name. 

Great choice!